README.md

```diff
@@ -20,6 +20,9 @@ schema file - `./source_data_schema.json`.
 ## Guide & Tips:
  
 - The team is interested in the approach you follow for solving this problem
+- Approach this problem as you would to build an end to end ETL pipeline.
+- Keep in mind that data in production almost always has anomalies, and a robust pipeline should account for and gracefully handle such situations.
```

Comment:
maybe give some examples of anomolies? Variable loads? corrupted or incomplete data rows